### PR TITLE
add footer for all pages

### LINKED ISSUE #17: Implement Footer.tsx
image

Location: **All Pages**
Number of Images: **2, _logo.webp_**
React Icon: **Yes**

Description: **This component should include all text, logo, and the 3 icons all centered vertically. The icon should be mapped using footerLinks.ts (see example). For now, use place holder link like google.com.  Both the logo and the icons are use like a button to bring the user to a specific links. When hovering on the logo, the opacity should be lowered. When the user click on the logo, the opacity should be lowered more, and it should bring them back to the home page. When hovering on join icons, the color should change from blue to white and the icon should be scaled down. When the user click on the icons, the opacity should be lowered, and it should bring them to the specific link using a "new tab".  Make sure to add duration for all transition of color/opacity**

PATH: 
1. /src/components/Footer.tsx
2. /src/data/footerLinks.ts    (I forgot to create a file for that so just create that yourself :) )

[Example for Footer](https://github.com/acm-ucr/robosub-website/blob/dev/src/components/Footer.jsx)
[Example of the Array for mapping the icons](https://github.com/acm-ucr/robosub-website/blob/dev/src/data/footerLinks.js)

### PR DESCRIPTION
**Pull request by minhhdtran**

### CODE REVIEW

**Comment by minhhdtran** [comment]
hearts_footer

File: tailwind.config.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ module.exports = {
     extend: {},
     colors: {
       hearts: {
+        white: "#FFFFFF",
```
Comment (qhgill) [inline tailwind.config.ts:8]: no need to add this, just use text-white or bg-white whenever you need

**Review by qhgill — CHANGES_REQUESTED** [review]
overall looks good, just change to use tailwind's default white to reduce redundancies

File: tailwind.config.ts (hunk 1 above)
Comment (qhgill) [reply tailwind.config.ts:8]: disregard, temporary fix for a problem I have to fix later

**Review by qhgill — APPROVED** [review]

**Pull request merged**